src/common.rs

```diff
@@ -410,6 +487,34 @@ named!(pub field_list<&[u8], Vec<Column> >,
        )
 );
 
+named!(field_value<&[u8], (Column,Literal) >,
+    chain!(
+        column: column_identifier_no_alias ~
+        multispace? ~
+        tag!("=") ~
+        multispace? ~
+        value: literal,
+        || { (column, value) }
+    )
+);
+
+named!(pub field_value_list<&[u8], Vec<(Column,Literal)> >,
+       many0!(
```

Comment:
Should this be a `many1!`? `field_value_list` is only used in the `UPDATE` statement, and I believe  the field-value list there cannot be empty, correct?

Reply:
Yes, you are right this should be `many1!` as stated here https://sqlite.org/lang_update.html